<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeWidgetForm</class>
 <widget class="QWidget" name="ThemeWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TelegramNews</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Стиль:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="themeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="animatedLabel">
       <property name="text">
        <string>Тема статей:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="animatedComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>animatedComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>themeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ThemeWidgetForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateUI()</slot>
 </slots>
</ui>
